moped-editor/src/components/LookupSelectComponent.js

```diff
@@ -24,10 +24,22 @@ const LookupSelectComponent = (props) => {
   return (
     <Select
       variant="standard"
-      style={{ minWidth: "8em" }}
       id={props.name}
       value={value || props.defaultValue}
       ref={ref}
+      disableUnderline
+      sx={{
+        minWidth: "8em",
+        padding: "10px",
+        fontSize: ".875rem",
```

Comment:
I don't think that I worded this well in the issue, but I think that the default 1rem font that DataGrid when a cell is in edit mode is great instead of making them all .875rem. Here is [my comment](https://github.com/cityofaustin/atd-moped/pull/1362#discussion_r1672721433) that was buried about what I saw in the MUI demo.

My line of thinking is that we can worry less about being consistent with these overrides as we continue on to other tables and let DataGrid handle the font sizing for us.

Reply:
ahhh you like the font being bigger on edit? Okay, ill revert back to the default!